### ISSUE TITLE
Create public dataset of parking transactions

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#7601]
I've staged a private dataset of Flowbird meter transactions [here](https://docs.google.com/presentation/d/1ByRWXmjHut88cTiPVOTgkwTJ2V5_m1Y_0j46FP_kdYw/edit#slide=id.p).

This is "Smartfolio" data from the DR-direct `transaction_history` report.

**Comment by johnclary** [comment]
@Charlie-Henry 
- Add descriptions to all of our parking datasets in Socrata
- [This will](https://data.austintexas.gov/dataset/Parking-Transactions/tag2-gnds) be the dataset.

**Comment by johnclary** [comment]
ETL is complete and deployed to Prefect test. The dataset is currently private.

@Charlie-Henry 
- please add a description and column metadata to public dataset. 
- Please review and delete [this](https://data.austintexas.gov/dataset/Parking-Transactions/tag2-gnds/revisions/0) dataset if it's not needed

**Comment by patrickm02L** [comment]
@johnclary wanted to get your thoughts on getting permissions to make this data set public. Charlie and I have talked to Joseph a few times, but he's hesitant release the data until the parking inventory is up to date.

**Comment by patrickm02L** [comment]
> ETL is complete and deployed to Prefect test. The dataset is currently private.
> 
> @Charlie-Henry
> 
> * please add a description and column metadata to public dataset.
> * Please review and delete [this](https://data.austintexas.gov/dataset/Parking-Transactions/tag2-gnds/revisions/0) dataset if it's not needed

6/22/22 talked in Product Sync and decided that these to-dos as well as providing a Read.me with caveats about the data quality and using data would get us to the point of publishing.

**Comment by patrickm02L** [comment]
In a meeting with SMS on 6/23/22 which included: Diana, Joseph, Patrick, Charlie, and Ryan, we discussed the use cases for publishing the data. 

Finance team has a need, requests from other internal teams were coming in, and there was an intention to use the data for staff to learn PowerBI. 

We determined that the order of operations for publishing the data would happen after Joseph and team have completed the parking inventory transfer which is a task that their new GIS analyst will work on once they come on board (start date 7/5/22).

In the meantime, we determined that internal requesters could be added as collaborators in order to get access to this private dataset. Everybody agreed this would be an easy fix. 

When SMS is in a good place with the parking inventory transfer, we'll set up a meeting to talk about how to responsibly release the data to the open data portal with the expectation that it would be rolled out with an announcement through PIO.
